COMMUNITY_PRESETS_GUIDELINES.md

```diff
@@ -47,6 +47,101 @@ Noise Reduction effect.
 - The folders above mentioned should contain a subdirectory with the name
 associated to the installed package. In example:
 - - `XDG_DATA_DIR/easyeffects/output/package-name`
+- The name of the package should be something descriptive e.g. `easyeffects-presets-LoudnessEqualizer`. If your package contains a group of presets without a particular "theme", naming a package `easyeffects-USERNAME-presets` is reasonable.
```

Comment:
Is this a reasonable change? Later on this doc gives examples like `wwmm-presets`. Putting the username in the package name makes sense to avoid collisions, but the username is also not very useful information as it doesn't explain what the package provides. 

At the very least this should doc be consistent, so if this lined isn't preferred it should be removed.

Reply:
It's OK. No problem.